lib/resolvers/error.js

```diff
@@ -31,14 +32,49 @@ const _cdsToGraphQLError = (context, err) => {
 }
 
 const _ensureError = error => (error instanceof Error ? error : new Error(error))
+const _clone = obj => Object.create(Object.getPrototypeOf(obj), Object.getOwnPropertyDescriptors(obj))
 
 const handleCDSError = (context, error) => {
   error = _ensureError(error)
-  // TODO: choose log level depending on type of error analogous to OData adapter
-  if (LOG_CDS._error) LOG_CDS.error(error)
+  _log(context.req, error)
   return _cdsToGraphQLError(context, error)
 }
 
+const _log = (req, error) => {
+  // do not log standard errors, e.g. TypeError
+  if (error.code === undefined && error.details === undefined) return
```

Comment:
Is there a reason we can't/shouldn't do this instead?
```suggestion
  if (!error.code && !error.details) return
```

Reply:
When it comes to code readability, I often avoid using negations — it's not a strict rule for every situation, but rather a guideline I follow in certain cases. My main reason for this approach is that code can sometimes be defined with a false value, such as an empty placeholder `''` or the number/code `0`. In this case, strict checks for `undefined` is preferable.